### ISSUE TITLE
emma_report enhancements

### ISSUE THREAD

**Issue by adammwilson** [AdamWilsonLab/emma_report#15]
A few ideas of easy things to add:

## Reports for individual parks:
1. add histogram of veld ages (of pixels within each park)
2. Add download links for a geotif (or vector geojson?) of veld ages for each park

@jslingsby @GMoncrieff  other ideas?

**Comment by bmaitner** [comment]
Some other ideas:

1) Total fire area in the park, e.g., in the preceding 12 months or something?
2) Veld age in each burned polygon from the preceding year? 
 (Link to CapeNature “key questions”)
3) Ryan suggested erosion risk based on slope and time since fire, but I'm not sure how easy this is.
4) I could generate a "delta NDVI" metric (current NDVI relative to that point one year ago) as a sort of placeholder for the NDVI anomaly data.

**Comment by bmaitner** [comment]
For the geotif/geojson of veld ages for each park, would it be useful to buffer this a bit? I imagine that fires  near the edge of the park may also be of interest?

**Comment by GMoncrieff** [comment]
This is obviously a rabbit-hole, but in the interest of rabbits:
https://github.com/SoilWatch/soil-erosion-watch
https://soilwatch.users.earthengine.app/view/soilerosionwatch

Erosion is probably something managers are VERY interested in, and relates very closely to post-fire veg recovery. We could just use this code (maybe swap out some layers)

**Comment by jslingsby** [comment]
Nicola and others at TMNP have been asking me about mapping soil erosion
after our recent big rains. Lots of landslides... Should be very
detectable, but it'd take a big landslide to trigger change in a MODIS
pixel... Probably more a Sentinel game...

**Comment by jslingsby** [comment]
Yes. Definitely buffer the park. "Neighbouring fuels" is a big concern to
managers...

**Comment by bmaitner** [comment]
> A few ideas of easy things to add:
> 
> ## Reports for individual parks:
> 1. add histogram of veld ages (of pixels within each park)
> 2. Add download links for a geotif (or vector geojson?) of veld ages for each park
> 
> @jslingsby @GMoncrieff other ideas?

Both of these have now been implemented in the reports.  For the histogram, I convert counts to area, which I thought might be more useful for management/reporting.  For the download links,  I provide a link to the geotif as well as links to a polygon version for each park (including a buffer of something like 10 km).
